content/port-binding.md

```diff
@@ -1,14 +1,52 @@
 ## VII. Port binding
+
 ### Export services via port binding
 
-Web apps are sometimes executed inside a webserver container.  For example, PHP apps might run as a module inside [Apache HTTPD](http://httpd.apache.org/), or Java apps might run inside [Tomcat](http://tomcat.apache.org/).
+#### 1. A twelve-factor app is completely self-contained and exports HTTP as a service by binding to a port.
+
+Web apps are sometimes executed inside a webserver container. For example, PHP
+apps might run as a module inside [Apache HTTPD](http://httpd.apache.org/), or
+Java apps might run inside [Tomcat](http://tomcat.apache.org/). **The
+twelve-factor app is completely self-contained** and does not rely on runtime
+injection of a webserver into the execution environment to create a web-facing
+service. Instead, the web app **exports HTTP as a service by binding to a port**
+and listening to requests coming in on that port.
```

Comment:
i note there is a typo here in the original text, I t should say "and listens to" or potentially "and **by** listening to"